pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -59,6 +57,26 @@
 # enable -h as additional help option
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
+if getattr(pywbem, 'PYWBEM_USES_REQUESTS', False):
```

Comment:
I propose comment here about PYWBEM_USES_REQUESTS, and why this code.

Reply:
DONE - added:
```
# Pywbem 1.0.0 changed to using the 'requests' package, which caused
# differences in how certificate checking is done. Pywbem introduced
# the pywbem.PYWBEM_USES_REQUESTS flag to indicate that. Since
# pywbemtools needs to work with pywbem versions before and after
# 1.0.0, we are checking the flag to generate the correct help message
# for each case.
```